### PR TITLE
Lower the maximum binding index in createBindGroupLayout to 640

### PR DESCRIPTION
**Pull request by nical**
See #3279

I suppose this should be properly discussed/approved at a standards meeting before being merged. In the mean time here is the proposed change.

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -5127,7 +5127,7 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
                         <div class=validusage>
                             - |this| is [=valid=].
                             - The {{GPUBindGroupLayoutEntry/binding}} of each entry in |descriptor| is unique.
-                            - The {{GPUBindGroupLayoutEntry/binding}} of each entry in |descriptor| must be &lt; 65536.
+                            - The {{GPUBindGroupLayoutEntry/binding}} of each entry in |descriptor| must be &lt; 640.
```
Comment (Kangz) [inline spec/index.bs:5130]: IIRC @kainino0x asked for a limit to be added for this value.

Comment (kainino0x) [reply spec/index.bs:5130]: Yes, I think we should, although it doesn't have to be in the same PR! We can leave the linked issue open until that's resolved.

**Comment by kainino0x** [comment]
Technically this is substantive though, so @nical can you link your github account to your w3c account and revalidate?
See https://labs.w3.org/repo-manager/pr/id/gpuweb/gpuweb/3312

Adding the limit can be in this PR or a followup one, whatever works for you.

**Review by kainino0x — APPROVED** [review]

**Comment by kainino0x** [comment]
Oh, I just saw @toji did the limit over in #3318. I'll close this then - sorry for the duplication!
You can still get your account linking set up for future contributions :)

**Comment by toji** [comment]
Apologies for not seeing that this had already been posted when I wrote mine. Thanks for the contribution!

**Pull request closed without merging**